jinahub/indexers/searcher/compound/FaissPostgresSearcher/faisspsql.py

```diff
@@ -39,19 +39,22 @@ def __init__(
         self.logger = get_logger(self)
 
         if total_shards is None:
-            self.total_shards = getattr(self.runtime_args, 'parallel', None)
+            self.total_shards = getattr(self.runtime_args, "parallel", None)
```

Comment:
seems like you missed the `-no-string-format` flag for black. Can you revert this?

Reply:
fixed